awswrangler/redshift.py

```diff
@@ -271,7 +271,7 @@ def _redshift_types_from_path(
     varchar_lengths: Optional[Dict[str, int]],
     parquet_infer_sampling: float,
     path_suffix: Optional[str],
-    path_ignore_suffix: Optional[str],
+    path_ignore_suffix: Union[str, List[str], None],
```

Comment:
Nitpick but what do you think ok `Optional[Union[str, List[str]]]` instead of using `, None`?

Reply:
Yap I would normally slightly prefer `Optional` but I noticed all the other functions with this parameter define it as `Union[str, List[str], None]` so I thought it would be good to make it consistent.

Reply:
Sounds good